lib/src/base-styles/_nx-page-header.scss

```diff
@@ -63,7 +63,7 @@
   }
 
   &:active, &.nx-page-header__link--current {
-    border-bottom-color: var(--nx-swatch-teal-60);
+    border-bottom-color: var(--nx-swatch-blue-40);
```

Comment:
And this `--nx-color-link`, double check with the designers.
Semantically, it sounds like it should match with `--nx-color-link` 🤔

Reply:
> And this `--nx-color-link`, double check with the designers. Semantically, it sounds like it should match with `--nx-color-link` 🤔

I did confirm with Jason, and changed here: [ad970fa](https://github.com/sonatype/sonatype-react-shared-components/pull/980/commits/ad970faf3b34c45366b877f1c9378b4370cfabcb)